source/Concepts.rst

```diff
@@ -34,7 +34,6 @@ The Core Stack Developer Concepts are much more detailed conceptual articles int
   Concepts/About-Build-System
   Concepts/About-Internal-Interfaces
   Concepts/About-Middleware-Implementations
-  Concepts/About-ROS-2-Client-Libraries
```

Comment:
Is this intended? I think this section is pretty important for ROS 2 users and therefore for Vulcanexus users.

Reply:
Thanks for the clarification. I did not realize that it was included previously (I only saw the snapshot that GitHub presented). Then I am OK with the changes once the relative path is applied.